cmake/FindJulia.cmake

```diff
@@ -0,0 +1,193 @@
+# This FindJulia.cmake from
+# <https://github.com/JuliaInterop/libcxxwrap-julia/blob/main/FindJulia.cmake>
+
+# Original FindJulia.cmake from <https://github.com/QuantStack/xtensor-julia-cookiecutter/
```

Comment:
Since this is an external file we are pulling in with a different (if somewhat confused license history), we should indicate that in our `LICENSE` file.

Reply:
Done.